app/main.py

```diff
@@ -1,6 +1,19 @@
 from typing import Callable
+from functools import wraps
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+
+    cache_dictionary = {}
+
+    @wraps(func)
+    def wrapper(*args) -> any:
```

Comment:
It would be better to use `Any` from `typing` for type annotation instead of `any` since it is not deprecated.